src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static removeDeadAnimals = () => {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  };
```

Comment:
While using a static method to remove dead animals is a valid approach, it's not necessary to call this method separately after each bite. Instead, consider filtering the `Animal.alive` array directly within the `bite` method after decreasing the health of the target. This would ensure that the array is always up to date without requiring an additional method call.